library/src/main/java/com/heinrichreimersoftware/materialintro/app/IntroActivity.java

```diff
@@ -1281,10 +1281,22 @@ public int getSlidePosition(Slide slide) {
     }
 
     @SuppressWarnings("unused")
+    public int getCurrentSlidePosition() {
+        return pager.getCurrentItem();
+    }
+
+    @SuppressWarnings("unused")
     public Fragment getItem(int position) {
         return adapter.getItem(position);
     }
 
+    @SuppressWarnings("unused")
```

Comment:
See #166 and 8757d41b120632243f84f2ebacfa9aedb77a77f0.
It's already implemented.